src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -208,7 +212,7 @@ export default class BrowserCell extends Component {
   }
 
   shouldComponentUpdate(nextProps, nextState) {
-    if (nextState.showTooltip !== this.state.showTooltip || nextState.content !== this.state.content ) {
+    if (nextState.showTooltip !== this.state.showTooltip || nextState.content !== this.state.content || nextState.showExecuteScriptDialog !== this.state.showExecuteScriptDialog) {
```

Comment:
Let's call this script in the config file and throughout code the confirmation dialog, see the config params suggestion in https://github.com/parse-community/parse-dashboard/issues/2474#issue-1771915187.

```
apps": [
  {
    "scripts": [
      {
        "title": "Delete Account",
        "classes": ["_User"],
        "cloudCodeFunction": "deleteAccount",
        "showConfirmationDialog": true,
        "confirmationDialogStyle": "critical" // or "info" for blue style
      }
    ]
  }
]
```

Reply:
ok changed this as well